### ISSUE TITLE
Error: No test scheduler initialized after upgrading to ng8.

### ISSUE THREAD

**Issue by damienwebdev** [ngrx/platform#1913]
## Minimal reproduction of the bug/regression with instructions:

```bash
git clone https://github.com/graycoreio/daffodil
cd daffodil
git checkout version_upgrade
npm install
ng build core && ng build product && ng test product
```
## Expected behavior:
I would expect not to get the following errors:

```
TypeError: expect(...).toBeObservable is not a function
Error: No test scheduler initialized
```

## Versions of NgRx, Angular, Node, affected browser(s) and operating system(s):
```
Node: 10.15.0
OS: darwin x64
Angular: 8.0.0
... animations, cdk, common, compiler, compiler-cli, core, forms
... language-service, platform-browser, platform-browser-dynamic
... platform-server, router, service-worker

Package                                    Version
--------------------------------------------------------------------
@angular-devkit/architect                  0.800.1
@angular-devkit/build-angular              0.800.1
@angular-devkit/build-ng-packagr           0.800.1
@angular-devkit/build-optimizer            0.800.1
@angular-devkit/build-webpack              0.800.1
@angular-devkit/core                       7.3.6
@angular-devkit/schematics                 7.3.6
@angular/cli                               8.0.1
@angular/pwa                               0.11.4
@ngtools/json-schema                       1.1.0
@ngtools/webpack                           8.0.1
@nguniversal/express-engine                8.0.0-rc.1
@nguniversal/module-map-ngfactory-loader   8.0.0-rc.1
@schematics/angular                        8.0.1
@schematics/update                         0.800.1
ng-packagr                                 5.2.0
rxjs                                       6.5.2
typescript                                 3.4.5
webpack                                    4.30.0
"@ngrx/effects": "^8.0.0-rc.0",
    "@ngrx/entity": "^8.0.0-rc.0",
    "@ngrx/router-store": "github:ngrx/router-store-builds",
    "@ngrx/schematics": "^8.0.0-rc.0",
    "@ngrx/store": "^8.0.0-rc.0",
    "@ngrx/store-devtools": "^8.0.0-rc.0",
```

## Other Information
This bug breaks all the described effect tests in the docs, as the same model of test no longer works

## I would be willing to submit a PR to fix this issue

[x] Yes (Assistance is provided if you need help submitting a pull request)
[ ] No

**Comment by Rlcolli4** [comment]
I am having the same problem, we ran `ng update @angular/cli @angular/core` and everything else worked with out an issue, but after running our unit tests, we saw all our effects tests failing. We get stack traces similar to the following.

<img width="1110" alt="Screen Shot 2019-06-04 at 11 16 11 AM" src="https://user-images.githubusercontent.com/12074293/58903459-559e7d80-86ba-11e9-85f0-4296475dc1dd.png">

I can provide information on version of angular if needed.

**Comment by Trent-Matthias** [comment]
Adding "target": "es5" to tsconfig.spec.json fixed it for me as a workaround.

**Comment by kekel87** [comment]
Hi, there is another workaround to keep "es2015": https://github.com/synapse-wireless-labs/jasmine-marbles/issues/40#issuecomment-499785967

It also link to this issue: https://github.com/synapse-wireless-labs/jasmine-marbles/issues/37

**Comment by brandonroberts** [comment]
Verified that the reproduction is fixed with the `0.6.0` release of `jasmine-marbles`.
